### PR TITLE
Feat: change dimmed style opacity for InnerBlocks

### PR DESCRIPTION
**Pull request by jbinda**
This PR change the opacity to 1 (so basically it turn off the dimming of unselected block). It's because latest decision about design. I keep the logic because we want to have possibility to bring it back easily and/or adjust the opacity

Please also refer to:
[Related gutenberg PR](https://github.com/WordPress/gutenberg/pull/21042)
[Columns PR](https://github.com/WordPress/gutenberg/pull/19013)
[Buttons PR](https://github.com/WordPress/gutenberg/pull/20191)

After this gets merged we should also update the sanity-test cases for [Group block](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/group.md#tc007)

To test:
- expect during navigation through nested structure that none of the block gets dimmed

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by jbinda** [comment]
Ci tests passed, however I wonder if I shouldn't add some note in `RELEASE-NOTES.txt` because it affects current production behaviour about dimming ? @pinarol wdyt ?

**Comment by pinarol** [comment]
Good point. It’d be good to add a note @jbinda

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 1.25.0
 ------
 * Improve icon on the "Take a Video" media option
+* Change dimmed style opacity to 1 for unselected block when navigating in nested structure
```
Comment (pinarol) [inline RELEASE-NOTES.txt]: Can we make it more easy to understand for external people/users?

```suggestion
* Removed the dimming effect on unselected blocks
```

**Comment by jbinda** [comment]
sure, done !

**Pull request merged**